pulser-core/pulser/result.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Classes to store measurement results."""
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from collections import Counter
+from dataclasses import dataclass
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from pulser.register import QubitId
+
+
+@dataclass  # type: ignore
+class Result(ABC):
+    """Base class for storing the result of a sequence run."""
+
+    atom_order: tuple[QubitId, ...]
```

Comment:
What is this variable `atom_order` used for, exactly? Is it some sort of qubit ordering?

Reply:
It's the qubit order in the bitstring. Since we don't enforce any type of convention on the qubit IDs, I figured this information is needed to avoid ambiguity.